easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0206-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'Bundle'
+
+name = 'XML-LibXML'
+version = '2.0206'
+
+homepage = 'https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod'
+description = "Perl binding for libxml2"
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
```

Comment:
May I ask why did you add `pkg-config`? I do not see it in the build log. 
```suggestion
```

Reply:
Ubuntu does not have pkg-config.  The build fails without pkg-config.
```eb XML-LibXML-2.0206-GCCcore.eb --rebuild
== temporary log file in case of crash /tmp/eb-RxAuVe/easybuild-o4oLky.log
== found valid index for /app/software/EasyBuild/4.3.2/easybuild/easyconfigs, so using it...
== found valid index for /app/software/EasyBuild/4.3.2/easybuild/easyconfigs, so using it...
== processing EasyBuild easyconfig /app/build/XML-LibXML-2.0206-GCCcore.eb
== building and installing XML-LibXML/2.0206-GCCcore-10.2.0...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== building...
== testing...
== installing...
== taking care of extensions...
== installing extension File::chdir 0.1010 (1/4)...
== installing extension Alien::Base 2.38 (2/4)...
== installing extension Alien::Libxml2 0.17 (3/4)...
== FAILED: Installation ended unsuccessfully (build directory: /app/build/XMLLibXML/2.0206/GCCcore-10.2.0): build failed (first 300 chars): cmd " perl Makefile.PL PREFIX=/app/software/XML-LibXML/2.0206-GCCcore-10.2.0 " exited with exit code 2 and output:
Could not find an appropriate pkg-config or pkgconf implementation, please install PkgConfig.pm, PkgConfig::LibPkgConf, pkg-config or pkgconf at /app/build/XMLLibXML/2.0206/GCCcore-10.2 (took 16 sec)

Reply:
Interesting, so it does use `pkg-config` and it can use it directly without the Perl module `PkgConfig`. Thanks :+1: